applications/surgical_scene_recon/train_standalone.py

```
@@ -0,0 +1,287 @@
+        args.training_iterations,
+        args.coarse_iterations,
+        not args.no_masks,
+    )
+
+    if checkpoint:
+        print(f"\n{'='*70}")
+        print("  SUCCESS!")
+        print(f"{'='*70}")
+        print(f"Trained checkpoint: {checkpoint}")
+        print("\nRun inference and render with:")
+        print(f"    ./holohub run surgical_scene_recon render")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)